model/vfs/fsck.go

```diff
@@ -111,6 +117,7 @@ type TreeFile struct {
 	IsDir    bool `json:"is_dir"`
 	IsOrphan bool `json:"is_orphan"`
 	HasCycle bool `json:"has_cycle"`
+	HasPath  bool `json:"-"`
```

Comment:
What does `"-"` mean here?

Reply:
It means that this field won't appear in the JSON serialization

Reply:
Thanks.